ensime-startup.el

```diff
@@ -117,6 +117,25 @@ that you have read this message.")
   (let ((ensime-prefer-noninteractive t))
     (ensime--maybe-update-and-start orig-buffer-file-name)))
 
+(defun is-dev-version? (version)
```

Comment:
an emacs lisp convention is `ensime-dev-version-p` (package prefix, `p` for "perchance")